### PR TITLE
Add tailwind CSS

### PR DESCRIPTION
**Pull request by whitep4nth3r**
Hello! So I wanted to a) try Tailwind and b) give people an easy way to add styles to the project without actually writing CSS!

I've styled the home page as a starter but left other stuff untouched.

I've added this to the README:

## Styling the front end

The project uses [tailwind CSS](https://tailwindcss.com/)!

System requirements:

```
Node.js
npm
```

## To install dependencies, run the following commands:

```
cd /HacktoberfestProject.Web
npm install
```

## Adding styles

The raw css is stored in `/HacktoberfestProject.Web/styles/raw.css`.

The aim of using tailwind, is this:

```
Instead of opinionated predesigned components, Tailwind provides low-level utility classes that let you build completely custom designs without ever leaving your HTML.
```

Ideally, we shouldn't have to add any custom css to raw.css. To follow this pattern, attempt to add and edit styles only in the HTML layout files.

## Compiling css

To compile any extra css you add to raw.css, run the following command:

```
cd /HacktoberfestProject.Web
npm run tailwind
```

Take a look at `package.json` to see what the `tailwind` task is doing. Essentially it takes the `styles/raw.css` file, processes it and moves the output to `wwwroot/css/output.css`.

Any questions, let me know!

### CODE REVIEW

**Comment by whitep4nth3r** [comment]
I haven't removed bootstrap completely yet - but this should be the aim.

**Review by CrypticEngima — APPROVED** [review]

**Comment by CrypticEngima** [comment]
Absolutly awesome work here @whitep4nth3r !! If any one wishes to see the development of this Pull Request the Vod is on twitch  [https://www.twitch.tv/videos/769089343](https://www.twitch.tv/videos/769089343)

File: HacktoberfestProject.Web/Views/Shared/_Login.cshtml [hunk 1]
```diff
@@ -1,10 +1,18 @@
-﻿<ul class="navbar-nav">
-	@if (!User.Identity.IsAuthenticated)
-	{
-		<a asp-action="Login" asp-controller="Auth" class="btn btn-outline-primary">Log In with GitHub</a>
-	} else
-	{
-		<a asp-action="Logout" asp-controller="Auth" class="btn btn-outline-primary">Logout</a>
-	}
+﻿@if (!User.Identity.IsAuthenticated)
```
Comment (rickvdbosch) [inline HacktoberfestProject.Web/Views/Shared/_Login.cshtml:1]: If the only difference for this if-statement is the text and location of the link, we could optimize and not have the svg in there twice. Downside: we have (something that looks like) an if-statement twice.

Pseudo-code:

```
<a ... asp-action="@(User.IsdAuthenticated ? "Logout" : "Login"") ... />
```

and

```
@(User.IsAuthenticated ? "Logout" : "Log in with GitHub")
```

**Review by rickvdbosch — APPROVED** [review]
I only have one comment, but that's also a possible optimization for a future PR 😃

**Comment by whitep4nth3r** [comment]
I wanted to do this but my logic handling in .NET is limited!

**Review by Layla-P — APPROVED** [review]
Good to go 💖🎉

**Pull request merged**